operator/src/main/java/org/bf2/operator/managers/IngressControllerManager.java

```diff
@@ -285,7 +289,12 @@ public void onDelete(IngressController obj, boolean deletedFinalStateUnknown) {
         // this is to patch the IngressController Router deployments to correctly size for resources, should be removed
         // after https://issues.redhat.com/browse/RFE-1475 is resolved.
         if (deploymentResourceBuilder.hasLimits() || deploymentResourceBuilder.hasRequests()) {
-            this.deploymentResourceRequirements = deploymentResourceBuilder.build();
+            this.azDeploymentResourceRequirements = deploymentResourceBuilder.build();
+
+            // use the default cpu for the kas/default ic
+            deploymentResourceBuilder.addToRequests(CPU, Quantity.parse("100m"));
```

Comment:
so this is default CPU you were mentioning in the comment?

Reply:
Yes, we're putting it in here explicitly so that it overwrites the 1700m we currently have set.  In a later release, we can remove this.